### PR TITLE
New types

### PR DESCRIPTION
**Pull request by bam241**
adding map<pair<string,string>, int> for @mbmcgarry :)

### CODE REVIEW

File: src/query_backend.h [hunk 1]
```diff
@@ -331,12 +331,22 @@ enum DbTypes {
...
+  MAP_PAIR_STRING_VL_STRING_INT, // ["std::map<std::pair<std::string, std::string>, int>, 4, [], ["MAP", ["PAIR", "STRING", "VL_STRING"], "INT"], false]
+  MAP_PAIR_VL_STRING_STRING_INT, // ["std::map<std::pair<std::string, std::string>, int>, 4, [], ["MAP", ["PAIR", "VL_STRING", "STRING"], "INT"], false]
+  MAP_PAIR_VL_STRING_VL_STRING_INT, // ["std::map<std::pair<std::string, std::string>, int>, 4, [], ["MAP", ["PAIR", "VL_STRING", "VL_STRING"], "INT"], false]
+  VL_MAP_PAIR_STRING_STRING_INT, // ["std::map<std::pair<std::string, std::string>, int>, 4, [], ["VL_MAP", ["PAIR", "STRING", "STRING"], "INT"], false]
+  VL_MAP_PAIR_STRING_VL_STRING_INT, // ["std::map<std::pair<std::string, std::string>, int>, 4, [], ["VL_MAP", ["PAIR", "STRING", "VL_STRING"], "INT"], false]
+  VL_MAP_PAIR_VL_STRING_STRING_INT, // ["std::map<std::pair<std::string, std::string>, int>, 4, [], ["VL_MAP", ["PAIR", "VL_STRING", "STRING"], "INT"], false]
+  VL_MAP_PAIR_VL_STRING_VL_STRING_INT, // ["std::map<std::pair<std::string, std::string>, int>, 4, [], ["VL_MAP", ["PAIR", "VL_STRING", "VL_STRING"], "INT"], false]
```
Comment (scopatz) [inline src/query_backend.h]: You didn't actually add HDF5 or SQLite to the list of backends that this type should be generated for.

**Comment by scopatz** [comment]
Seems reasonable, waiting for CI after update.

**Comment by bam241** [comment]
@scopatz is it good to go ?

File: src/query_backend.h (hunk 1 above)
Comment (scopatz) [inline src/query_backend.h]: Hi @bam241, you still need to add HDF5 and SQLite to these data types. See the above examples.

**Comment by bam241** [comment]
@scopatz this should be ok now!

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**